packages/amplify-category-auth/src/provider-utils/awscloudformation/import/index.ts

```diff
@@ -1255,6 +1268,13 @@ export const headlessImport = async (
     answers.userPool = await cognito.getUserPoolDetails(resolvedEnvParams.userPoolId);
   } catch (error) {
     if (error.name === 'ResourceNotFoundException') {
+      if (projectHasAuth(context, false)) {
+        await context.amplify.removeResource(context, AmplifyCategories.AUTH, answers.resourceName, { headless: true });
```

Comment:
are we 100% sure that `ResourceNotFoundException` refers to imported resource?
what if `ResourceNotFoundException` is a result of manual actions?

Reply:
By manual actions, you mean userPool is manually deleted ?

Reply:
yes, but then I checked upstream logic and it seems we're checking that we're dealing with imported auth.